spec/support/service_spec_helper.rb

```diff
@@ -11,6 +11,10 @@ module ServiceSpecHelper
     end
   end
 
+  def has_permissions(perms)
```

Comment:
This is a small nit but maybe this should be something like `permissions_exist`? `has_permissions` to me seems to indicate that these permissions are attached to some object, but in reality you're just setting these up to be in the database, right?

Reply:
The name is supposed to be declarative. e.g. this object (now) `has_permissions(perms)`.

What if I renamed it to `apply_permissions(perms)`?

Reply:
What object, though? You're just passing a list of permissions and it's creating them in the database, but it's not assigning them to any object, right?